hack/local-up-cluster.sh

```diff
@@ -1043,7 +1043,8 @@ function parse_eviction {
 function install_cni {
   echo "Installing CNI plugin binaries ..." \
     && curl -sSL --retry 5 --output /tmp/cni."${CNI_TARGETARCH}".tgz "${CNI_PLUGINS_URL}" \
-    && echo "${CNI_PLUGINS_AMD64_SHA256SUM}  /tmp/cni.amd64.tgz" | tee /tmp/cni.sha256 \
+    && CNI_PLUGINS_SHA256SUM="CNI_PLUGINS_${CNI_TARGETARCH^^}_SHA256SUM" \
```

Comment:
you can follow the kind scripts to fix this
https://github.com/kubernetes-sigs/kind/blob/c45ee4a90254d86c9e06c227cc0ac0f557e8f9c8/images/base/Dockerfile#L173-#L177